packages/picasso/src/Select/utils/flatten-options/flatten-options.ts

```diff
@@ -0,0 +1,13 @@
+import { Option, OptionGroups } from '../../types'
+import { isOptionsType } from '../../utils'
+
+const flattenOptions = (options: Option[] | OptionGroups): Option[] => {
+  if (isOptionsType(options)) {
+    return options
+  }
+
+  return Object.values(options)
+    .reduce((result: Option[], optionList) => [...result, ...optionList], [])
```

Comment:
```suggestion
    .reduce((result: Option[], optionList) => result.concat(optionList))
```

it is 3 times faster